### PR TITLE
DOCS/RTD: Update home page content

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -41,34 +55,38 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
     $ ./contrib/configure-release --prefix=$PWD/install
     $ make -j8 install
 
+    # find the first RDMA-capable network interface with IPv4 address
+    $ ipaddr=$(ls -d /sys/class/net/*/device/infiniband | cut -d/ -f5 | \
```
Comment (mike-dubman) [inline docs/source/index.rst]: lets remove it

File: docs/source/index.rst [hunk 2]
```diff
@@ -41,34 +55,38 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
...
+               xargs -n1 ifconfig | awk '/inet / {print $2}' | head -1)
+
+    $ gcc test/examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
+          -Iinstall/include -Linstall/lib
+    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ ./ucp_client_server &
+    $ ./ucp_client_server -a ${ipaddr}
```
Comment (mike-dubman) [inline docs/source/index.rst]: lets replace ${ipaddr} with
```<ipaddr-for-rdma-capable-network-interface>```

File: docs/source/ucx_features.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Runs on virtual machines (using SRIOV) and containers (docker, singularity).
+- Can utilize either MLNX_OFED or Inbox RDMA drivers.
+- Tested on major Linux distributions (RedHat/Ubuntu/SLES).
+
+GPU support
+***********
+- Cuda (for NVIDIA GPUs)
+- ROCm (for AMD GPUs)
+
+Protocols, Optimizations, and Advanced Features
```
Comment (mike-dubman) [inline docs/source/ucx_features.rst]: no need comma before and

File: docs/source/ucx_features.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Automatic selection of best transports and devices.
+- Zero-copy with registration cache.
+- Scalable flow control algorithms.
+- Optimized memory pools.
+- Accelerated direct-verbs transport for Mellanox devices.
+- Pipeline protocols for GPU memory
+- QoS and traffic isolation for RDMA transports
+- Platform specific optimizations (such as memcpy)
+- Multi-rail support
+- Bare-metal, containers and cloud environments support
```
Comment (mike-dubman) [inline docs/source/ucx_features.rst]: - advanced protocols for transfer messages of different sizes

**Comment by yosefe** [comment]
@mike-dubman your comments were applied

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 08-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/ad59c4c240d1dae2bbf59bad45e859db948991e8/_main) | :question: ABORTED
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/a67cc93e4ea6b996ac6dfc1bbca96c05886702ab/hpc-test-node-new_W0) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/d79ef168e10b5e8fcbb369634783d6ae9b700ff4/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/bb370af08976b102faa379e7955fa24ca9b007be/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/59584327d8ccbbcaf5323dfc040d6575c6a852f9/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/8ecedfeabfbad650103489e69fb1e87384e183d6/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/07c281b0ea3af421761d038b7585329d3305e723/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/a17c02513786f4954d05f7701c290ea8620cea5e/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/7e334e6e67b97b6e5a331de6127e0258f4dcb2da/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/277d3ffc7de3dc46cd60de56bb25e38237abe68e/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/b024d4684e028b9a44120d4f32a7b3a6e853c0bc/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/f792baef9019ae4686254100f0f7f6f0d817103d/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/39ac75f7ab1311191642de2a7dc7fef98876c0c1/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/f1ed7f157a312f1adb16e71af23d92e70815e361/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/8425115327da8298bf717039e3b215b96f971788/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/3e5b286a536a05a203e99e3ff21fac6e693ef9b1/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/5faa8e3c99b6a4650cf70c1efdc172958145fa3c/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/9496b35d0bb76a35c14833989bc6a873f0b06653/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/7116a9ac910d882fe719d7d3941e70690339ee84/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/f1921506f53cc0e2e3944dd4fd7ee8dc2167c989/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/7acc5b77c8d83425783130514077709922fe03d5/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/b1d0a9031c53d865dbc9f0a1c7b08f70c23f11d7/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/aa9380ed7681bc7d4f627828ebffd5886e565cea/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/b0870041d708bb5d6f9bce7884c3f67c19237fc2/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/a7ea5857e1843eefed63f15f2c2d16a8/raw/efd802dc963a1c1c53e0e7d6a352280058a9e2df/hpc-test-node-new_W1) | :question: UNKNOWN

File: docs/source/index.rst [hunk 5]
```diff
@@ -41,34 +55,34 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
     $ ./contrib/configure-release --prefix=$PWD/install
     $ make -j8 install
 
+    $ gcc test/examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
+          -Iinstall/include -Linstall/lib
+    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ ./ucp_client_server &
+    $ ./ucp_client_server -a <ip-addr>   # <ip-addr>: Address of a local RoCE or IPoIB interface
```
Comment (mike-dubman) [inline docs/source/index.rst]: s/Address/IP Address

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 4 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 08-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/7e13667ffe08799c5d110c58c483855268bd528a/_main) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/b006211f0ff32c7b55a1fdb33eb74711d9a32438/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/b063300ce86f2bf3a9d55ace5ce4633b572a88bd/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/f7cf022a9d074ceaa2170beaea35489fa08ca278/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/2fc0ff66743258f3f19e4f5b9f01d8a9083aa2e1/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/f502eff8cbf99feb0f2e8d22e526851a6b7a7161/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/1d0d9ddc5213dda98d2f868e8576c74e67780138/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/ff39bc5783f013b22b025993132b1947aebc7710/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/75573699504064d484d72cf10b98919b119c08c4/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/d3a40782ef1046f816c1243f7074b70f91d45239/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/d1b26b4475076287a1bedc60d6135b1a209c9f5f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/14b1d8451718d2062da2ef79e43c6babdfd30a38/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/65adefe0a388247c5081c1c7a177072e0305fae1/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/9a26f04cfc7041f74f0782d443541f821a19a5e8/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/7f1472a4ff8b5d483694708c61537f4e8ad4ddd5/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/1147b83ccbbe7d4c168873986c36db59e9890d16/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/e645a37ea9af39e987e5251f997eb183c1136cf3/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/e0a42c69490c52ae5419b7ff1fc53a30630b6d1a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/a9e97731373640894dddb03e70060605b8caeb0c/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/dc01c8b2d28c49febe4e5aaae057ea26fd377cd2/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/3aeb11f8c70acdef1dad748c29fddf911dae0f7f/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/8ff055fc871558ecdb81e2f5f3934f6f9aebe6fa/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/1ff122ac6b522dbb7a9e7321a4b49af7220a6b02/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/25e2183d3f5e018c3efb8ca83181777f26e2d387/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3d3d885318626cdb533bb9e4b93ec73e/raw/5b604d53e815f6713ec08af8a353bbfa06261a78/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 25 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 08-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/bab5ce6c246540e71c90dafe72980dd7eae10cc2/_main) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/05213bef0e24b0a8cfd83855d90a599f8a3ef070/hpc-arm-cavium-jenkins_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/9909821bffb243b8e3d2a311d43c1f34e8173937/hpc-arm-cavium-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/752bdf853980a4162633b19c3febb3ea6f11c1d2/hpc-arm-cavium-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/a0e6a0adcadcd0c2bf89101aa530942ad1e5bf5c/hpc-arm-cavium-jenkins_W3) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/73006db70699cc49282868c4446d74d024f1c89f/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/a0ef2ce5b5c0bc282b5eb0bbd209af876922736a/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/fa9b91a34ec21d706fd6b3b5d7df7650669c19c1/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/1117bafb9a69778bd45eb32e0ea1524ee3e19dd1/hpc-arm-hwi-jenkins_W3) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/7085f9f2627bc69daf829737fd9a99e9e2ac5500/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/cec01cdde06f5cc612217bc32fa48a5ee62ed31d/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/3c879d3dba73bb4d710611e3e7ae0c0703e8075f/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/6f29142cfa1ff01c2db79b45e09a4569c7e64390/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/440ecd6793220b0e68b9c38a3a4ef68d64d48e08/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/2c75cc4d80dd6c5a8e31baec47998d3517af4e97/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/6cb192e732776ba11edbdc50d3a2076544084926/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/b248ab9196b793569639a863a170bb4c20676290/hpc-test-node-legacy_W3) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/e9f5a32b40eb356083c4a1ce93c582ef4fb01cff/hpc-test-node-new_W0) | :x: FAILURE
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/8d7e241a367ecbd72d11b4da9559bdb0a59a711c/hpc-test-node-new_W1) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/0c50408a9d7cc8eb82be75b2cd221cec03accfad/hpc-test-node-new_W2) | :x: FAILURE
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/4696704743275997d278df9d69ea0fc7fc49c688/hpc-test-node-new_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/fb5629de1dc2f3677cf116692a26eb189848ebd2/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/49578289280f65d1758f105a6ee2d102f3926e8e/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/e0ee9875c6b5cb52501f02bbf655881c97808641/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a711836c0ffff2f1e88b10921a22ecac/raw/69467dba22d4ef22b29e2ed53491cf236ba717cf/r-vmb-ppc-jenkins_W3) | :x: FAILURE

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 08-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/2b20b7228a962e38ab7e6884dae977196a17b047/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/150d51b380a22f2d4f9f78dbf7ec28bcba4c1af1/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/4529d04f5da0c3e937adbe2eda12753254d74782/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/78fb08521bd389704cb9d5358a8ea45a1489594a/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/b7d540479bc51e2c850a025ff953983c6d2bf821/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/a38ba5c0be74f4196c19f4fa5922161aec5aa966/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/6066fe1f428f8062b9c07654f9ad2377a129a466/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/f03101f850b7643b993a761ed84aebfe2e99cdf4/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/9ccfd7beb572867b064af599bc56b10a637e5949/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/fdcb893652f7fc1323341e7787fc179230897921/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/039291742d12acbab17b974b81d3e6a5f95082bc/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/524d610444d15d6fe7360cb886018fbde5edf0ef/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/8f5fd5bb7713912a9577088cf8e753559b78f7b2/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/0f2f5455a63f769af6a3eee3d9771027b51c8c93/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/b20db7189e26d94b0fc2ce3214c0b9ae8446ee2e/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/b052e255c91af083c1e07049fd4596a4982023f0/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/842fae9f2bd84dc5aa287488470de524cea13db0/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/58984e0e835e5d49c8febf329487f05ca2c6fab7/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/5c9a3a91444e9a0e46f4d58222e18305ac4dc6f0/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/329a3601f653d2d3ef7f8286bb73fa7cd97ef634/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/3fe072409f9a3f31cc148d241a1f0dd0c7686ba4/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/8bec6a5b8fb51b86bbce3d3f7b8d64a11928d81f/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/6f8a6415242bc5dd6eaf50ef682c86405f01fed3/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/6bd83c04c80886bd95e70a68337b40d5c1a6fead/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b8309feb6d4577427532a6534ad2a1e8/raw/9e663c72b53afa8abae43b3957fb1b72a98431de/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
fedora30 vm issue
bot:pipe:retest

**Comment by yosefe** [comment]
@shamisp can you pls take a look?

**Comment by yosefe** [comment]
merging, per offline discussion with @shamisp . test failures are irrelevant and due to setup issue.

**Comment by shamisp** [comment]
@tonycurtis @bbenton - general ask, can you please review the new user documentation texts. It is largely based on wiki. thanks !

**Pull request merged**